src/WWTMVC5/RedisConfig.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Configuration;
+
+namespace WWTMVC5
+{
+    /// <summary>
+    /// Delegates the configuration of the Redis Cache to use another app setting
+    /// This centralizes our redis connection string and allows use in session offloading
```

Comment:
Can you add that this is called by the session defined in web.config? Since it's not directly referenced, it may appear to be dead code.